.github/workflows/html.yml

```diff
@@ -3,16 +3,18 @@
 name: HTML Doc
 
 on:
-  push:
-    branches:
-      - master
+  pull_request:
```

Comment:
If we are going to support forks, then we probably want to use `pull_request_target` instead of `pull_request`

Reply:
Do I need to include both?

Reply:
Don't need to conclude both.